Source_Code.txt

```diff
@@ -0,0 +1,48 @@
+def makePowerup():
+  pic=makePicture(pickAFile())
+  picHeight=getHeight(pic)
+  #getting the picture height helps alignment
+  
+  canvasHeight=500
+  #the canvas is square so the width is the same as the height
+  canvas=makeEmptyPicture(canvasHeight,canvasHeight, black)
+ 
+  show(canvas)
+  
+  targetX=(canvasHeight-picHeight)/2
+  targetY=targetX
+  #make pic centered
+  
+  number=4
+  #Number of times to move up and down
+  
+  for loop in range (0,number):
+  #make image move up then down
+    maxHeight=10
+    #number of pixels move
+    
+    for up in range (0,maxHeight):
+      copyInto(pic, canvas, targetX, targetY-up)
```

Comment:
If I used an image that wasn't the power up lighting one it would get an error on this line